src/reduced_basis/rb_eim_evaluation.C

```diff
@@ -292,10 +280,8 @@ Number RBEIMEvaluation::get_eim_basis_function_value(unsigned int basis_function
                                                      unsigned int comp,
                                                      unsigned int qp) const
 {
-  if (basis_function_index >= _local_eim_basis_functions.size())
-  {
-    libmesh_error_msg("Invalid basis function index: " + std::to_string(basis_function_index));
-  }
+  libmesh_error_msg_if(basis_function_index >= _local_eim_basis_functions.size(),
+                       "Invalid basis function index: " + basis_function_index);
```

Comment:
You dropped the std::to_string() here.

Perhaps more alarmingly, this made it through CI!?!  Even the --enable-paranoid-warnings --enable-werror runs!?!

It looks like ```-Wstring-plus-int``` is a clang warning but not yet a gcc warning, and we only use --enable-werror on gcc builds.  Right now we're in "we'll take what we can get" mode on CI, but after that's straightened out we ought to just turn on --enable-paranoid-warnings --enable-werror on every CI build.

Reply:
Ha, I meant to change the` +` into a `<<`, I'll push that now...